gradio/helpers.py

```diff
@@ -690,6 +690,26 @@ def special_args(
         elif type_hint == routes.Request:
             if inputs is not None:
                 inputs.insert(i, request)
+        elif (
+            type_hint == Optional[oauth.OAuthProfile] or type_hint == oauth.OAuthProfile
```

Comment:
Or `oauth.OAuthProfile | None`

Reply:
Handled in https://github.com/gradio-app/gradio/pull/4943/commits/103fba5ad947f14870b2a0934f5701c0625740eb